packages/app-builder/src/locales/en/auth.json

```
@@ -25,5 +25,8 @@
   "email-verification.wrong_place": "Wrong place ? go back to <SignIn>{{signIn}}</SignIn>.",
   "reset-password.send": "send reset password email",
   "reset-password.email_sent": "An email has been sent to you with a link to reset your password.",
-  "reset-password.wrong_place": "Wrong place ? go back to <SignIn>{{signIn}}</SignIn>."
+  "reset-password.wrong_place": "Wrong place ? go back to <SignIn>{{signIn}}</SignIn>.",
+  "sign_up.read_more": "Read more :",
+  "sign_up.first_connection_guide": "First Connection Guide",
+  "sign_up.warning.db_not_ready": "Your database is not initialised."
```

```suggestion
  "sign_up.warning.instance_not_initialized": "You have not created a first organization and user."
```
for what it's worth, I'd suggest something more along those lines for the message (or something more precise, depending on the respective values of both flags)